### PR TITLE
SMHE-2068: Object config SetSpecialDaysCommandExecutor

### PR DESCRIPTION
**Pull request by wdevries97**
Implemented ObjectConfig for the SetSpecialDaysCommandExecutor

### CODE REVIEW

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json [hunk 1]
```diff
@@ -1223,6 +1223,25 @@
         }
       ]
     },
+    {
+      "tag": "SPECIAL_DAYS_TABLE",
+      "description": "Special Days Table",
+      "note": null,
```
Comment (stefanermens) [inline osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: If there is no note, then the field can be removed.

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json [hunk 2]
```diff
@@ -1223,6 +1223,25 @@
...
+      "version": 0,
+      "obis": "0.0.11.0.0.255",
+      "group": "ABSTRACT",
+      "meterTypes": ["SP","PP"],
+      "attributes": [
+        {
+          "id": 2,
+          "description": "entries",
+          "datatype": "array",
+          "valuetype": "DYNAMIC",
```
Comment (stefanermens) [inline osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: DYNAMIC means that the value can be updated by the meter itself. That is not the case here. SET_BY_CLIENT is the correct valuetype.

**Review by stefanermens — CHANGES_REQUESTED** [review]

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json (hunk 1 above)
Comment (wdevries97) [reply osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: Removed the field

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json (hunk 2 above)
Comment (wdevries97) [reply osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: Changed to SET_BY_CLIENT

**Review by stefanermens — APPROVED** [review]

**Pull request merged**